README.md

```diff
@@ -63,26 +80,32 @@ Download the docker-compose.yml, the .env and the setup.sh files:
 
 #### Create a docker-compose environment
 
-- Edit the `.env` to your liking. Adapt this file with your FQDN.
-  - This file supports only simple `VAR=VAL` lines (see [Documentation](https://docs.docker.com/compose/env-file/)).
-  - Don't quote your values.
-  - Variable substitution is *not* supported (e.g. `OVERRIDE_HOSTNAME=$HOSTNAME.$DOMAINNAME`).
+- The `.env` file contains the default environmental variables and their explanations. You can also find them in the more readable file [ENV.MD](ENV.MD). These variables will be passed to `docker-compose.yml` during running. You can change these variables in `docker-compose.yml`.
 - Install [docker-compose](https://docs.docker.com/compose/) in the version `1.7` or higher.
 
-#### Start Container
-    docker-compose up -d mail
-
-#### Create your mail accounts
+#### Create your mail accounts (at least one account)
 
     ./setup.sh email add <user@domain> [<password>]
 
 #### Generate DKIM keys
 
     ./setup.sh config dkim
 
-Now the keys are generated, you can configure your DNS server by just pasting the content of `config/opendkim/keys/domain.tld/mail.txt` in your `domain.tld.hosts` zone.
+Now the keys are generated, you can configure your DNS server by just pasting the content of `config/opendkim/keys/domain.tld/mail.txt` in your `domain.tld.hosts` zone if you are running your own DNS server. Or, go to your DNS provider website and use the content to create a record (or something similar):
+- Record type: `TXT`
+- Main record: `mail._domainkey`
+- Record value: 
+```
+( "v=DKIM1; h=sha256; k=rsa; "
+	  "p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxG6b8/BgftnqyTnC30RvoEiN1BhCxP...+A0fA0SN/c9"
+	  "bP7yYSOOfeRWWDmi4rpbdsorzrUcnfDLTm8oTlRETYc3pGaDXk3kuVnJ4P5O9bxiCXR/Zs7t8/ywuBwcc..." )  ; ----- DKIM key mail for mydomain.org
+```
+Also create an `A` record to point `mail.youdomain.tld` to your machine public IP address.
+
+#### Start Container
+    docker-compose up -d mail
 
-#### Restart and update the container
+#### Restart and update the container if necessary
 
     docker-compose down
     docker pull tvial/docker-mailserver:latest
```

Comment:
the `docker pull` part is not necessary. instead suggest to do `docker-compose pull`